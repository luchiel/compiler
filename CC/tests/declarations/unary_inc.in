void main() { int a = 5; ++a; }