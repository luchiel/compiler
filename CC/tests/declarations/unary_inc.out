type <int>
type <float>
type <void>
type <int*>
	pointer to <int>
function main
	returns type <void>
	arguments
		<no args>
	{{}}
		variable a of type <int> initialized by
			{int 5}
				type <int>
	| 
	+-{;}
	  | 
	  +-{++x}
				type <int>
	    | 
	    +-{a}
					type <int>
